.travis.yml

```diff
@@ -6,7 +6,7 @@ rust:
 sudo: false
 install:
 - git clone https://github.com/xiph/speexdsp.git
-- cd speexdsp && ./configure --prefix=$HOME/installed_speexdsp && make && make install &&
+- cd speexdsp && make && make install &&
```

Comment:
You should run `autogen.sh` and then do the `./configure && make` part probably...

(There is a reason why I'd rather go with the first minimal set in :))

Reply:
Yah will try that. Last attempt probably, else will just test without sys for first run